lib/mayaUsd/fileio/primUpdaterManager.cpp

```diff
@@ -619,8 +616,7 @@ bool pushCustomize(
               }
 
               auto dstPath = makeDstPath(dstRootParentPath, srcPath);
-              auto updater
-                  = createUpdater(ufePulledPath, srcLayer, srcPath, dstLayer, dstPath, context);
+              auto updater = createUpdater(srcLayer, srcPath, dstPath, context);
```

Comment:
Didn't need the ufePulledPath any more, and dstLayer was unused, so removed them.